client/tailwind.config.js

```diff
@@ -49,6 +52,8 @@ module.exports = {
       inherit: 'inherit',
       current: 'currentColor',
       transparent: 'transparent',
+      LinkText: 'LinkText',
+      ButtonText: 'ButtonText',
```

Comment:
Should we be opting for `link-text` as the key here, happy to go with `LinkText` - don't mean to be a pain but just checking as we are setting up new conventions here.
OR
`system-link-text` (longer but clearer?)